api-tests/features/p2-no-photo-id.feature

```diff
@@ -0,0 +1,435 @@
+@Build
+Feature: P2 no photo id journey
+  Rule: Experian KBV
+    Background: Start P2 no photo id with Experian KBV
+      Given I activate the 'm2bBetaExperianKbv' feature set
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'end' event
+      Then I get a 'page-ipv-identity-postoffice-start' page response
+      When I submit an 'end' event
+      Then I get a 'prove-identity-no-photo-id' page response
+      When I submit an 'next' event
+
+    Scenario: P2 no photo id journey - Experian - Happy path
+      Then I get a 'claimedIdentity' CRI response
+      When I submit 'kenneth-current' details with attributes to the CRI stub
+        | Attribute | Values         |
+        | context   | "bank_account" |
+      Then I get a 'bav' CRI response
+      When I submit 'kenneth' details to the CRI stub
+      Then I get a 'nino' CRI response
+      When I submit 'kenneth-score-2' details with attributes to the CRI stub
+        | Attribute          | Values                                      |
+        | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":2} |
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-pre-experian-kbv-transition' page response
+      When I submit a 'next' event
+      Then I get a 'kbv' CRI response
+      When I submit 'kenneth-score-2' details with attributes to the CRI stub
+        | Attribute          | Values                                          |
+        | evidence_requested | {"scoringPolicy":"gpg45","verificationScore":2} |
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+
+    Scenario: P2 no photo id journey - Experian - BAV dropout:
```

Comment:
We don't get to the KBV so it probably doesn't matter, but the end-to-end tests used the HmrcKbv flag for BAV and NINO dropout.
Given that we don't get to the KBV stage in these tests, maybe move them to the `dropout` section below?

Reply:
Thank you for the feedback. I understand enabling a flag when not needed isn't ideal but I'm struggling to piece together how the re-categorisation would work. I am keen to keep as much as possible in the two main rule groupings Experian and KBV just to make it easier reading. The other rule groups are there because they need to delve into a specific state really. With that, I will merge what's here, and if anything comes up, I can take another look.